smarty/sampler_smirky.py

```diff
@@ -835,7 +845,7 @@ def sample_types(self):
         env = random.choice(proposed_envList)
 
         # determine create or destroy:
-        if random.random() < 0.5:
+        if random.random() < 0.2:
```

Comment:
Is there a way we can determine what this ought to be? (not for this PR, just curious how to think about it...)